### PR TITLE
zombie gamemode semi rework

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
we have revs at home

- IIs get syndie radio implants to communicate with eachother and form a plan
- once the event starts, you have to wait 5 minutes before being able to zombify.
  every single person whos played zombies knows of the random musician turning in med while you tried to do a cool plan with other IIs, now you are forced to communicate!
- changes the flavortext to be more of a bioterrorist than someone who happened to show up to work infected with romerol, why would they suddenly want to kill people instead of getting help / isolating? since it says syndicate sponsored the radio implant makes sense
- if sec overhears the random group of people chanting DEATH TO NANOTRASEN! in maints before the delay ends they can kill/gib/cremate/etc the body and it wont get to become a zombie :trollface:

this only affects the zombie gamemode and the admin antag control, the event is still relatively disorganized

## Why / Balance
zombies is one of the worst gamemodes, this should make it better for everyone (even the crew get a bit more time to have a normal shift instead of instant med biting -> evac)

## Technical details
add DelayedRuleComponent for adding the zombie components after a delay

## Media
it real

## Requirements
- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
no

**Changelog**
:cl:
- tweak: Initial Infecteds now get radio implants to communicate thanks to the Syndicate.
- tweak: Initial Infecteds now have to wait 5 minutes before they can turn, form a plan!

### CODE REVIEW

**Comment by Avalon-Proto** [comment]
Now *this* is how to do a zombie mode

**Comment by ThataKat** [comment]
Unfortunate to lose the "sick and doesn't realize" rp potential but that hardly happened anyways. This seems like a good change to me

**Comment by deltanedas** [comment]
you can still do that for the survival zombies since theres no syndicate nature to it

File: Content.Server/DeltaV/GameTicking/Rules/DelayedRuleSystem.cs
Comment (MilonPL) [inline Content.Server/DeltaV/GameTicking/Rules/DelayedRuleSystem.cs]: doesn't need a nullable warning suppression, `.Value` checks for null anyways

**Review by MilonPL — CHANGES_REQUESTED** [review]

**Comment by Unkn0wnGh0st333** [comment]
make it lore wise that IIs are suicide operatives from gorlex (or something) with an engineered version of romerol that's slower and can activate on command made by interdyne.

**Comment by deltanedas** [comment]
they are already syndicate affiliated and IIs have always been able to trigger zombification with the action

**Comment by Unkn0wnGh0st333** [comment]
> they are already syndicate affiliated and IIs have always been able to trigger zombification with the action

Loooore wise, the important stuff is always in the details... or maybe I'm just one of those nerds that looks to deep into item descriptions.

**Comment by Avalon-Proto** [comment]
Soooo whens this gonna be done

**Comment by deltanedas** [comment]
still waiting for admin approval

**Comment by therealDLondon** [comment]
> still waiting for admin approval

You have it. Lets see how this rolls

**Review by MilonPL — APPROVED** [review]

**Pull request merged**
